proto/frontend/src/device_mgr.cpp

```diff
@@ -639,10 +639,11 @@ class DeviceMgrImp {
       switch (finfo->match_type) {
         case PI_P4INFO_MATCH_TYPE_VALID:
           {
-            auto valid = mf->mutable_valid();
+            auto exact = mf->mutable_exact();
```

Comment:
Can you add some comments?  I assume this is for backward compatibility, bit it would help to clarify.

Reply:
@teverman Done. PTAL.